docusaurus/docs/iOS/customization/localization.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Translation and i10n
+---
+
+The SDK offers you in-house localization for all strings contained in UI components. For our internal localization, we provide `strings` file for each language that we translate. For advanced plural localization cases (typing indicator showing how many people are typing or how many responses are in thread), we use `stringsdict` with `NSStringPluralRuleType`. 
+
+## The Change of language on the device.
+
+Language for the StreamChat is changed together language of the device or simulator. See [how to change device language](https://support.apple.com/en-us/HT204031).
+There are however some workarounds how you can enforce the application to use only specific languages or let the users choose.
+
+## Adding new language which is not present in StreamChat
+
+StreamChatUI uses `strings` and `stringsdict` files to handle localize the framework.
+`Appearance` structure contains closure `localizationProvider`, which is responsible for deriving `localizedString` from exact bundle. 
+By default, this bundle is the  `StreamChatUI` framework bundle, but you can customize it to anything you need in 3 steps:
+
+1. If you don't have yet your `strings` or `stringsdict` files, you need to create those new files in your project/target, preferably `Localizable.strings` and `Localizable.stringsdict`.
```

Comment:
Change to:

1. If you don't have `strings` or `stringsdict` files in your project, add those new files to `Localizable.strings` and `Localizable.stringsdict`.

Reply:
Done.